.github/pull_request_template.md

```diff
@@ -0,0 +1,50 @@
+<!--
+     For Work In Progress Pull Requests, please use the Draft PR feature,
+     see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
+
+     For a timely review/response, please avoid force-pushing additional
+     commits if your PR already received reviews or comments.
+
+     Before submitting a Pull Request, please ensure you've done the following:
+     - 👷‍♀️ Create small PRs. In most cases this will be possible.
+     - ✅ Provide tests for your changes.
+     - 📝 Use descriptive commit messages.
+     - 📗 Update any related documentation and include any relevant screenshots.
+-->
+
+## What type of PR is this? (check all applicable)
+
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Chore
+- [ ] Refactor / Optimization
+- [ ] Documentation Update
+
+## Description
+
+## Related Tickets & Documents
+
+<!--
+For pull requests that relate or close an issue, please include them
+below.  
+For example having the text: "closes #1234" would connect the current pull
+request to issue 1234.  And when we merge the pull request, Github will
+automatically close the issue.
+-->
+
+- Related Issue #
+- Closes #
+
+## QA Instructions, Screenshots, Recordings
+
+[ Optional ] Please replace this line with instructions on how to test your changes, as well as any relevant images.
+
+
+## Added/updated tests?

```

Comment:
Is adding a testing section necessary for prs?

most Rust errors are compile time errors, and the "does it compile" test captures those.

I've seen testing being necessary for unsafe rust, but there is no unsafe rust in this code-base.

Reply:
`There should be system tests

Reply:
Also, if someone makes a change something or adds a new feature instructions on how to check that the change is working correctly in-game makes reviewing PRs easier